tests/browser/test_webelem.py

```diff
@@ -360,189 +360,191 @@ def test_multiple_levels(self):
                  stubs.FakeChildrenFrame(second[2:4])]
         root = stubs.FakeChildrenFrame(first)
         children = webelem.get_child_frames(root)
-        self.assertEqual(len(children), 7)
-        self.assertIs(children[0], root)
+        assert len(children) == 7
+        assert children[0] is root
         for frame in [root] + first + second:
-            with self.subTest(frame=frame):
-                frame.childFrames.assert_called_once_with()
+            frame.childFrames.assert_called_once_with()
 
 
-class IsEditableTests(unittest.TestCase):
+class TestIsEditable:
 
     """Tests for is_editable."""
 
-    def setUp(self):
+    @pytest.yield_fixture(autouse=True)
+    def setup(self):
+        old_config = webelem.config
         webelem.config = None
+        yield
+        webelem.config = old_config
+
+    @pytest.yield_fixture
+    def stub_config(self, stubs):
+        """Fixture to create a config stub with an input section."""
+        config = stubs.ConfigStub({'input': {}})
+        with mock.patch('qutebrowser.browser.webelem.config', new=config):
```

Comment:
That must have been a brain fart from my part. Fixed! :smile:

Completely agree that for tests converted to pytest style we should use `monkeypatch` (for simple substitutions) or `pytest-mock` (when you need the power of `Mock` objects).